apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.tsx

```diff
@@ -406,15 +406,17 @@ export const Confirmation: React.FC = () => {
       {modalVisible && (
         <Modal
           title={
-            confirmSignatureResponse?.httpStatusCode > 400
-              ? 'Auðkenning tókst ekki'
+            confirmSignatureResponse?.httpStatusCode >= 400
+              ? confirmSignatureResponse?.code === 7023
```

Comment:
Consider creating an enum for dokobit error codes

Reply:
Just commented for now.